src/Routes/authentication.routes.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express")
+const authRoute = express.Router()
+const authController = require("../controllers/authentication.controller")
+
+
+/**
+ * @swagger
+ *  components:
+ *      schemas:
+ *          Login:
+ *             type: object
+ *             properties:
+ *                 email:
+ *                     type: string
+ *                 password:
+ *                      type: string
+ */
+
+
+
+/**
+ * @swagger
+ * /auth/login:
+ *  post:
+ *      summary: Login for Users
+ *      tag:
+ *          - auth  
+ *      description: Login for User
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: '#/components/schemas/Login'
+ *      responses:
+ *          200:
+ *              description: Login Successfully
+ *              
+ */ 
+
+authRoute.post('/login', authController.login)
+
+
+module.exports = {authRoute}
```

Comment:
The auth route..i am not highlight every new file or file change just the main or important ones as the whole structure was changes.
authRoute, doctorRoute,patientRoute and general route are made with there respective controllers in seprate files,